src/Pages/EventManager/EventManagerModal.js

```diff
@@ -203,6 +203,7 @@ function EventManagerModal(props) {
             <Label>Event Description</Label>
             <Input
               type='textarea'
+              maxLength = {80}
```

Comment:
okay. is the height of the card hardcoded bc this seems to be taking care of long descriptions?